src/interceptors/response.ts

```diff
@@ -348,6 +348,7 @@ export function defaultResponseInterceptor(axios: AxiosCacheInstance): ResponseI
 
         return {
           cached: true,
+          stale: true,
```

Comment:
you can't you use .cached here? since this boolean will always be true when cached is true too?

Reply:
I see your point. In other cases stale may be undefined/false when cached is true.

Reply:
Are there other cases where cache is true and the state is stale?

Reply:
What I meant was:

why `response.cached` cannot be used in your case? Since both these values will always have the same value?

Reply:
They will be the same when staleIfError is set, but is other cases `response.stale` may be falsy when `response.cached` is true. For example: when a request is made, and the cache is still valid (i.e. now() < ttl), then the cache is returned with `{...repsonse, cached: true, stale: false}`. Does that make sense?

Reply:
@arthurfiorette, I added more tests in cases where `cached: true` and `stale: false` to show where `stale` won't always have the same value as `cached`. This might make it a little more clear. Please review and lmk what you think.

Reply:
Hi @arthurfiorette, just following up on this since it's been a while. Let me know if you have further questions.